### PR TITLE
Fixes getting setup

### PR DESCRIPTION
**Pull request by seabbs**
I decided to switch to working with Rstudio this week to get the participant experience. Did the getting set up workflow and made some corrections. 

Made the choice to switch to `pak` as may save some issues with people getting setup.

### CODE REVIEW

**Comment by seabbs** [comment]
I'm now trying to run the sessions and finding that none of the stan models will compile due to not being able to find the include functions

File: getting-set-up.qmd [hunk 1]
```diff
@@ -23,7 +23,7 @@ The course relies on running stan through the `cmdstanr` **R** package, which it
 This requires a separate installation step:
 
 ```{r cmdstan_install, eval = FALSE}
-cmdstanr::install_cmdstan()
+cmdstanr::install_cmdstan(cores = 2)
```
Comment (sbfnk) [inline getting-set-up.qmd:26]: why 2?

Comment (seabbs) [reply getting-set-up.qmd:26]: so that is compiles faster and under the assumption they will have at least two we can use?

**Comment by seabbs** [comment]
> Given that it works in the CI, is this an RStudio specific issue?

No idea!

**Comment by seabbs** [comment]
Bug is unrelated to the course. It looks like the issue with spaces in file paths has come up again (I was working out of a google drive folder called My Drive - moving the folder fixes it.

**Comment by sbfnk** [comment]
That's really good to know (especially as it may well affect course participants).

I wonder if it's something we could raise/fix upstream at some point.

**Comment by seabbs** [comment]
So what is super weird is they have fixed it (i.e there is a closed issue from a year ago) so it is super weird it has really appeared with all new installs of everything.

File: getting-set-up.qmd (hunk 1 above)
Comment (sbfnk) [reply getting-set-up.qmd:26]: minor point of course but I'm not sure why we'd overrule the `cmdstanr` default here. Also, every extra argument adds complexity so I think we should only use what's definitely necessary (i.e. not this).

Comment (sbfnk) [inline getting-set-up.qmd:26]:
```suggestion
cmdstanr::install_cmdstan()
```

Comment (seabbs) [reply getting-set-up.qmd:26]: the stan default is 1 core and so slow is the reason. People new to cmdstan won't know it exists. I don't really mind either way

Comment (seabbs) [reply getting-set-up.qmd:26]: I don't really understand why we would want people to have to wait longer when installing but I also don't really mind

**Review by sbfnk — APPROVED** [review]

**Pull request merged**